protocol/avdl/keybase1/teams.avdl

```diff
@@ -1369,7 +1369,8 @@ protocol teams {
   }
 
   // @emailInviteMessage is an argument used as a welcome message in email invitations sent from the server
-  TeamAddMembersResult teamAddMembersMultiRole(int sessionID, TeamID teamID, array<UserRolePair> users, boolean sendChatNotification, union { null, string } emailInviteMessage);
+  // @defaultChannelsOverride is ConversationIDs to add the members to instead ot the team wide settings. if unset, use team wide setting
```

Comment:
```suggestion
  // @defaultChannelsOverride is ConversationIDs to add the members to instead to the team wide settings. if unset, use team wide setting
```

Reply:
I thought the list here would just be what the service should add the user to. Instead of being unset can the GUI just echo back the list of defaults it already has?

Reply:
It could here, but it's used in other places that don't know what the default channels are. The defaults should also apply in places where the gui won't be calling anything, like when someone accepts an invite. So here, an override to tell whatever is universally handling these to add to a different set if that makes sense